src/Context/ApiContext.php

```diff
@@ -169,8 +169,8 @@ public static function restore($fileName = self::FILENAME_CONFIG_DEFAULT)
     /**
      * @param $fileName
      *
-     * @return string[][]
      * @throws BunqException
+     * @return string[][]
```

Comment:
1. It's an purpose PSR-5 standard by the [Framework Interop Group](http://www.php-fig.org/).
 (https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#719-throws)

1. It's the default inside the CodeStyle fixer itself. There are some request for configuration but for now it's not possible. (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1602)

1. The phpDocumentor docs are showing the same examples as the PSR-5 purpose.

I must honestly admit, I don't know better than this is the 'default', interesting to read how you guys think about it. So what will it be?, are we going left or right? In other words, will I undo or implement this fix?